core/templates/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -180,8 +180,14 @@ angular.module('oppia').directive('questionPlayer', [
           ctrl.getActionButtonIconHtml = function(actionButtonType) {
             var iconHtml = '';
             if (actionButtonType === 'BOOST_SCORE') {
-              iconHtml = '<img class="action-button-icon" src="' +
-              getStaticImageUrl('/icons/rocket@2x.png') + '"/>';
+              iconHtml = `<picture>
+              <source type="image/webp" 
+              src="${getStaticImageUrl('/icons/rocket@2x.webp')}">
+              <source type="image/png" 
+              src="${getStaticImageUrl('/icons/rocket@2x.png')}">
+              <img class="action-button-icon" src=" +
+              ${getStaticImageUrl('/icons/rocket@2x.png')} + "/>
+              </picture>`;
```

Comment:
```suggestion
              src="${getStaticImageUrl('/icons/rocket@2x.png')}">
              <img class="action-button-icon" src="
              ${getStaticImageUrl('/icons/rocket@2x.png')}"/>
              </picture>`;
```

Reply:
Looks like this one hasn't been addressed yet.